docs/admin/customize.rst

```diff
@@ -116,11 +114,11 @@ static files served to clients.
 Custom quality checks and auto fixes
 ------------------------------------
 
-You have implemented code for :ref:`custom-autofix` or :ref:`custom-checks` and
-now it's time to install it into Weblate. First place them into your Python
-module with Weblate customization (see :ref:`custom-module`). Then enabled it
-is just matter of adding its fully-qualified path to Python class to
-appropriate settings (:setting:`CHECK_LIST` or :setting:`AUTOFIX_LIST`):
+To install your code for :ref:`custom-autofix` or :ref:`custom-checks` and in Weblate:
+
+1. Place the files in your Python module containing Weblate customization (see :ref:`custom-module`).
+2. Add its fully-qualified path to the Python class in the dedicated settings
```

Comment:
Not sure about this. (Or the other instance below). They point to different ":setting"s too…

Reply:
There is different setting for checks, fixes and addons. Maybe all these could be squashed to one chapter as they are samoe for most of the things?

Reply:
Saw you editing, my attempt was:

> Custom quality checks, -addons and auto-fixes
---------------------------------------------

To install your code for :ref:`custom-autofix` or :ref:`custom-checks` and in Weblate:

1. Place the files in your Python module containing the Weblate customization
   (see :ref:`custom-module`).
2. Add its fully-qualified path to the Python class in the dedicated settings
   (:setting:`WEBLATE_ADDONS`), (:setting:`CHECK_LIST` or :setting:`AUTOFIX_LIST`):

.. code-block:: python

  CHECK_LIST = (
      'weblate_customization.checks.FooCheck',
  )
  
  or
 
   WEBLATE_ADDONS = (
      'weblate_customization.addons.ExamplePreAddon',
   )

.. seealso::

   :ref:`own-checks`, :ref:`own-addon`, :ref:`addon-script`

.. _custom-addon-modules: